docs/src/reference/the-traversal.asciidoc

```diff
@@ -1991,6 +2009,22 @@ g.V(1).properties().label()
 
 link:++https://tinkerpop.apache.org/javadocs/x.y.z/core/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.html#label--++[`label()`]
 
+[[length-step]]
+=== Length Step
+
+The `length()`-step (*map*) returns the length incoming string traverser. Null values are not processed and remain as null when returned.
```

Comment:
what does `length()` do with non-string types?

Reply:
An `IllegalArgumentException` will be thrown (now added in the docs)